homework-9/server.js

```diff
@@ -0,0 +1,85 @@
+var halls = ['hall1', 'hall2', 'hall3', 'hall4', 'hall5'];
```

Comment:
Loša je praksa popuniti array objektima koje se onda kasnije mijenja ili ako uopće niti ne pripadaju tom array-u. Na kraju imaš u arrayu i ove stringove a i objekte što array čini ne iskoristivim.
Praksa je da array ima samo jednu vrstu objekta u sebi